### ISSUE TITLE
"Unicode error: no space left" cause strange behaviors

### ISSUE THREAD

**Issue by ConteZero** [lihop/godot-xterm#55]
While executing some commands the following errors appear in the logs:
```
Unicode error: no space left
Unicode error: invalid skip
```
Some texts are not displayed in the terminal and sometimes strange behaviors appear on the screen.
[test-2022-08-09.zip](https://github.com/lihop/godot-xterm/files/9294238/test-2022-08-09.zip)

I've uploaded a zipped mp4 file because github do not recognize it as valid mp4 file.
I've used Godot 3.5 and `man scout` is not the only command that has this problem, in my system it happens with other commands too, here are some examples:
```
man -Len -Tutf8 virt-edit
man -Len -Tutf8 dialog
man -Len -Tutf8 systemd-xdg-autostart-generator
man -Len -Tutf8 firewall-cmd
```

**Comment by lihop** [comment]
@ConteZero I can confirm the "Unicode error" messages and that they occur when texts are not displayed:

![issue3](https://user-images.githubusercontent.com/3696783/184029604-476b517b-5494-4d22-ad0f-f2a86cb71798.gif)

I'm still digging into the cause of these errors and trying to find a way to reproduce them reliably. Some of the other strange behavior in the video you posted (shown below) can be explained by a new issue I've created #56 (Incorrect intial PTY size).

![2022-08-11-100644_802x460_scrot](https://user-images.githubusercontent.com/3696783/184028854-07036a28-f6e5-4c42-9e72-1e1d3857eaa2.png)

**Comment by ConteZero** [comment]
I can replicate the error with this file:
[log_man_virt-edit.txt](https://github.com/lihop/godot-xterm/files/9311607/log_man_virt-edit.txt)

It is the result of this command `man -Len -Tutf8 --nh --nj virt-edit > log_man_virt-edit.txt` and if I execute `cat log_man_virt-edit.txt` I get the error on the log and the output is truncated.

I think the problem is the • character that appear the first time on line 294.
If I add more text after line 294 the problem keeps popping up, but if I add 2 or more characters before that line the problem disappears (with only 1 character the problem remains).

**Comment by lihop** [comment]
@ConteZero I think I've fixed this issue in 3ad0e10 and issue #56 in f774c90. I have released a beta with these patches along with patches for issues #51 and #53.

Could you install [version 2.1.1-beta1](https://github.com/lihop/godot-xterm/releases/tag/v2.1.1-beta1) and let me know if all the issues you have reported are fixed? I don't know how you installed GodotXterm, but the GDNative code has changed, so you will need to recompile the binaries or download the new ones from the releases page.

If everything is working as expected and you have no more issues to report, then I plan to go ahead and release bugfix-version 2.1.1.

Thanks for your feedback so far, it is much appreciated.

**Comment by ConteZero** [comment]
I did some tests and I can confirm that all issues are solved for me.

With version 2.1.1-beta1 I get the following error on the log console when the project is loaded but it does not appear to cause problems when you run it:
`res://addons/godot_xterm/pty.gd:105 - Invalid get index 'cols' (on base: 'Control (terminal.gd)').`
